### PR TITLE
Consolidate gpperfmon sql

### PR DESCRIPTION
**Pull request by cjcjameson**
Across versions, separate files had been created. Going forward, there
will be just one gpperfmon.sql for setting up the gpperfmon database.

### CODE REVIEW

File: gpAux/gpperfmon/src/gpmon/gpperfmon_install [hunk 1]
```diff
@@ -120,26 +120,9 @@ if __name__ == '__main__':
...
-    cmd = Command("PGPORT=%d psql -f %s/lib/gpperfmon/gpperfmon3.sql template1" % (options.port, gphome))
-    commands.append(cmd)
-    
-    # 4.x gpperfmon schema additions
-    cmd = Command("PGPORT=%d psql -f %s/lib/gpperfmon/gpperfmon4.sql gpperfmon" % (options.port, gphome))
-    commands.append(cmd)
-
-    # 4.1 gpperfmon schema additions
-    cmd = Command("PGPORT=%d psql -f %s/lib/gpperfmon/gpperfmon41.sql gpperfmon" % (options.port, gphome))
+    cmd = Command("PGPORT=%d psql -f %s/lib/gpperfmon/gpperfmon.sql template1" % (options.port, gphome))
```
Comment (melanieplageman) [inline gpAux/gpperfmon/src/gpmon/gpperfmon_install]: Are all of these going to create tables in the template1 database?

Comment (Chibin) [inline gpAux/gpperfmon/src/gpmon/gpperfmon_install]: > Are all of these going to create tables in the template1 database?

Good point, fortunately we have a `\c gpperfmon` in the sql, so we should be covered.

File: gpAux/gpperfmon/src/gpmon/gpperfmon.sql [hunk 2]
```diff
@@ -0,0 +1,660 @@
+--  !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```
Comment (larham) [inline gpAux/gpperfmon/src/gpmon/gpperfmon.sql]: If you are touching everything, consider sorting table sql herein by alpha, for easier manipulation later. In other words, "create table public.system_history"  would come after all the queries_* tables, etc.

Comment (cjcjameson) [inline gpAux/gpperfmon/src/gpmon/gpperfmon.sql]: Good point -- I would consider that a good need-to-have as a separate commit on this before being mergeable.

File: gpAux/gpperfmon/src/gpmon/gpperfmon_install (hunk 1 above)
Comment (asimrp) [inline gpAux/gpperfmon/src/gpmon/gpperfmon_install]: It's unusual and scary to use `template1` like this.  Can we run `createdb gpperfmon` followed by `psql -f ... gpperfmon`?

Comment (Chibin) [inline gpAux/gpperfmon/src/gpmon/gpperfmon_install]: > It's unusual and scary to use template1 like this. Can we run createdb gpperfmon followed by psql -f ... gpperfmon?

That's my initial thought as well (looks scary), but thought `\c gpperfmon` would suffice.
Will separate it out to be more explicit and safer.

**Comment by asimrp** [comment]
Thank you for removing `template1`.  LGTM.

**Pull request merged**